src-built-in/components/titleBar/TitleBar.jsx

```diff
@@ -0,0 +1,54 @@
+/*!
+* Copyright 2017 - 2020 by ChartIQ, Inc.
+* All rights reserved.
+*/
+import React from "react";
+import { useTitleBar } from "@chartiq/finsemble-ui/react/hooks";
+import { AlwaysOnTopButton, DockingButton, LinkerButton, ShareButton, initializeTitleBar, TabRegion, CloseButton, MaximizeButton, MinimizeButton,  } from "@chartiq/finsemble-ui/react/components";
+import "../../../assets/css/finsemble.css";
+import "../../../assets/css/_titleBar.css";
+
+/**
+ * This is the Title Bar component, which is rendered at
+ * the top of every Finsemble-controlled HTML window.
+ * You can customize this template by adding or removing
+ * elements and sytling as you see fit.
+ * 
+ * The visibility of each of the controls is controlled by
+ * config. For example, setting the
+ * "foreign.components.Window Manager.showLinker" property
+ * to false will hide the <LinkerButton/>.
+ */
+const TitleBar = () => {
+	const {
+		showLinkerButton,
+		showShareButton,
+		showDockingButton,
+		showAlwaysOnTopButton,
+		showMinimizeButton,
+		showMaximizeButton,
+		showCloseButton
+	} = useTitleBar();
+	return (
+		<>
+			<div className="fsbl-header-left">
+				{showLinkerButton && <LinkerButton />}
+				{showShareButton && <ShareButton />}
+			</div>
+			<div className="fsbl-header-center">
+				{/* If tabbing is disabled, <TabRegion/> will
+					only display the title */}
+				<TabRegion/>
+			</div>
+			<div className="fsbl-header-right">
+				{showDockingButton && <DockingButton />}
+				{showAlwaysOnTopButton && <AlwaysOnTopButton/>}
+				{showMinimizeButton && <MinimizeButton />}
+				{showMaximizeButton && <MaximizeButton />}
+				{showCloseButton && <CloseButton />}
+			</div>
+		</>
+	);
+}
+
+initializeTitleBar(TitleBar);
```

Comment:
can you remind me why this initialization logic couldn't be wrapped in TItleBarShell?

Reply:
The reason is that it injects the DOM to which the titlebar is mounted. Without it, where would we mount? The conclusion we reached over call was to rename to `initializeTitlebar` to `mountTitlebar`.